model/registry/ios.yaml

```
@@ -0,0 +1,44 @@
+groups:
+  - id: registry.ios.lifecycle.events
+    prefix: ios
+    type: attribute_group
+    brief: >
+        This document defines attributes that represents an occurrence of a lifecycle transition on the iOS platform.
```

```suggestion
        The iOS platform on which the iOS application is running.
```

nit: To make it consistent with the Android description